decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] Great job so far  👍 you are almost there. Kindly add the `attr_accessor :nameable` to the decorator class so that a user would be able to access the `@nameable` instance variable 👍